builders/interact/scaffold-eth.md

```diff
@@ -0,0 +1,376 @@
+---
+title: Using Scaffold-ETH
+description: You can deploy a Solidity DApp with a React UI and subgraph on Moonbeam in minutes by using Scaffold-ETH. Learn how in this tutorial.
+---
+
+# Using Scaffold-ETH to Deploy a DApp on Moonbeam
+
+![Scaffold-ETH Banner](/images/builders/interact/scaffold-eth/scaffold-eth-banner.png)
+
+## Introduction {: #introduction } 
+
+[Scaffold ETH](https://github.com/scaffold-eth/scaffold-eth){target=blank} is a collection of commonly used Ethereum development tools to quickly deploy a Solidity smart contract, and launch a DApp with a React frontend and a deployed subgraph. There are premade templates for common DApp types such as NFT’s, ERC20 tokens, multi-sig wallets, simple DEXs, and so on.
+
+Scaffold-ETH consists of several sub-components, including Hardhat, The Graph and a React UI. All of these components can be used on Moonbeam networks with some slight modifications. This guide will walk through the steps to deploy and run the default sample contract and DApp that Scaffold-ETH comes with on a Moonbeam network. 
+
+## Checking Prerequisites {: #checking-prerequisites } 
+
+--8<-- 'text/common/install-nodejs.md'
+
+To run The Graph component of Scaffold-ETH, you also need to have the following installed on your system to run a local The Graph node from Docker:
+
+ - [Docker](https://docs.docker.com/get-docker/){target=blank}
+ - [Docker Compose](https://docs.docker.com/compose/install/){target=blank}
+ - [JQ](https://stedolan.github.io/jq/download/){target=blank}
+
+### Installing Scaffold-ETH {: #installing-scaffold-eth } 
+
+First, we need to download Scaffold-ETH from its GitHub repository. 
+
+From the command line, enter:
+
+```
+git clone https://github.com/scaffold-eth/scaffold-eth.git
+```
+
+After the download completes, run:
+
+```
+yarn install
+```
+
+![Scaffold-ETH installation output](/images/builders/interact/scaffold-eth/scaffold-eth-1.png)
+
+Once the dependencies have been installed without any errors in the console output as in the above screenshot, we can proceed to modifying the different components of Scaffold-ETH. 
+
+## Modify Configurations {: #modify-configurations }
+
+We need to make modifications to the configurations of the three major components that make up Scaffold-ETH.
+
+### Hardhat Component {: #hardhat-component }
+
+Let's begin with making some modifications to the Hardhat component under the `/packages/hardhat` folder.
+
+1. The file we need to modify is `scaffold-eth/packages/hardhat/hardhat.config.js`. Under the `module.exports/networks` section, we want to add the network definitions for the Moonbeam network(s) that we would like to use, including the RPC endpoint, the chain ID, and the contract deployment account's private key. 
```

Comment:
For all of the numbered list items throughout, no period is required at the end of the sentences